File: INICIANTE/JAVASCRIPT/1060.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var contador = 0

for(i=0;i<6;i++){
    var x = prompt("")
    if(x > 0){
    contador = contador + 1
    }
}
console.log(contador + " valores positivos")

File: INICIANTE/JAVA/1060.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int contador =	0;

        for(int i=0;i<6;i++){
            double x = sc.nextDouble();
            if(x > 0){
            contador ++;
            }
        }
        System.out.println(contador + " valores positivos");
        sc.close();
    }
}
